examples/asr/conf/hat/conformer/conformer_hat_bpe.yaml

```diff
@@ -0,0 +1,267 @@
+# It contains the default values for training a Conformer-Hybrid_Autoregressive_Transducer ASR model, large size (~120M) with Transducer loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+# Here are the recommended configs for different variants of Conformer-Transducer, other parameters are the same as in this config file.
+#
+#  +--------------+---------+---------+----------+------------------+--------------+--------------------------+-----------------+
+#  | Model        | d_model | n_heads | n_layers | conv_kernel_size | weight_decay | pred_hidden/joint_hidden | pred_rnn_layers |
+#  +==============+=========+========+===========+==================+==============+==========================+=================+
+#  | Small   (14M)|   176   |    4   |    16     |       31         |     0.0      |           320            |        1        |
+#  +--------------+---------+--------+-----------+------------------+--------------+--------------------------+-----------------+
+#  | Medium  (32M)|   256   |    4   |    16     |       31         |     1e-3     |           640            |        1        |
+#  +--------------+---------+--------+-----------+------------------+--------------+--------------------------+-----------------+
+#  | Large  (120M)|   512   |    8   |    17     |       31         |     1e-3     |           640            |        1        |
+#  +--------------+---------+--------+-----------+------------------+--------------+--------------------------+-----------------+
+#  | XLarge (644M)|  1024   |    8   |    24     |        5         |     1e-3     |           640            |        2        |
+#  +--------------+---------+--------+-----------+------------------+--------------+--------------------------+-----------------+  
+
+# You may find more info about Conformer-Transducer here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-transducer
+# Pre-trained models of Conformer-Transducer can be found here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/results.html
+# The checkpoint of the large model trained on NeMo ASRSET with this recipe can be found here: https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_conformer_transducer_large
+
+name: "Conformer-Transducer-BPE"
```

Comment:
Rename to Conformer-HybridAutoregressiveTransducer-BPE

Reply:
How about  Conformer-HAT-Transducer-BPE?

Reply:
Is `Conformer-HAT-BPE` OK? `Conformer-HybridAutoregressiveTransducer-BPE` looks too long. `Conformer-HAT-Transducer-BPE` has two Transducer words because "T" in "HAT" means "Transducer".

Reply:
Yep sounds great